src/node_jsvmapi_types.h

```diff
@@ -34,4 +36,48 @@ struct napi_property_descriptor {
   void* data;
 };
 
+enum napi_valuetype {
+  // ES6 types (corresponds to typeof)
+  napi_undefined,
+  napi_null,
+  napi_boolean,
+  napi_number,
+  napi_string,
+  napi_symbol,
+  napi_object,
+  napi_function,
+};
+
+enum napi_typedarray_type {
```

Comment:
You mean [like this](https://github.com/nodejs/abi-stable-node/pull/88/files#diff-57762ba270aadd394d6acdf9f06841a9R111) ?